pkg/suite/suite.go

```diff
@@ -130,12 +133,76 @@ func (s *suite) Run(t *testing.T, suiteName string) {
 		Expect(err).NotTo(HaveOccurred())
 		state.SetFramework(framework)
 
-		// Setup client for connecting to WC
-		cluster, err := framework.LoadCluster()
+		cb, err := clusterbuilder.GetClusterBuilderForContext(mcContext)
```

Comment:
it'd be really nice to skip cluster creation when a existing cluster is provided.

Reply:
Ah, yeah! It should do the same as `cluster-test-suites` where it checks for env vars being set first and if not then create a new one. I'll update it.

Reply:
Updated